contextualized/regression/tests.py

```diff
@@ -104,7 +183,11 @@ def test_naive(self):
             },
             link_fn=LINK_FUNCTIONS["identity"],
         )
-        self._quicktest(model)
+        self._quicktest(model, dataclass_type="dataloader")
+        self._quicktest(model, dataclass_type="datamodule", pred_dl_type="test")
+        self._quicktest(model, dataclass_type="datamodule", pred_dl_type="train")
+        self._quicktest(model, dataclass_type="datamodule", pred_dl_type="val")
+        self._quicktest(model, dataclass_type="datamodule", pred_dl_type="full")
```

Comment:
We shouldn't run 4 convergence tests just to try one new prediction set. It would be better to do all of the prediction tests after a single convergence run.